www/js/app.js

```diff
@@ -1664,4 +1664,13 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         });
     }
 
+    
+    
+    document.addEventListener('keydown' , (event)=>{
```

Comment:
@KaranKaira Because we still support some old browsers, including IE11, you can't use the arrow notation. Please refactor this function and be sure to test that your code works on IE11 as a baseline. Note that we're not using Babel or Core JS, so code currently has to be written in ES5. Also, please ensure you use consistent spacing (look at examples of how we use spacing with if statements, and avoid adding extraneous spaces before comma separators.

Reply:
Sure.

Reply:
Done.